MSAL/src/public/MSALDefinitions.h

```diff
@@ -157,6 +157,27 @@ typedef NS_ENUM(NSUInteger, MSALDeviceMode)
     MSALDeviceModeShared
 };
 
+/**
+ Platform SSO status on macOS device
+ */
+typedef NS_ENUM(NSUInteger, MSALPlatformSSOStatus)
+{
+    /*
+        Administrator hasn't configured Platform SSO in sso config.
+    */
+    MSALPlatformSSONotEnabled,
+    
+    /*
+     Administrator has configured Platform SSO in sso config. But device has not been registred with ADRS via platform SSO
```

Comment:
ADRS doesn't mean much externally, so maybe replace it with "AAD"?

Reply:
ok sure